v3/jaykeeb/aumz_work/aumsworkhotswap.json

```diff
@@ -0,0 +1,183 @@
+{
+  "name": "Aums Work Hotswap Version",
```

Comment:
`Aums` or `Aumz`?

Probably dont need `Version`?

Reply:
sorry, aumz is right. yes dont need version